File: responses/KT623X-42Q/reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(X):
    # write your code in Python
    a = X
    r = 0
    if a < -1:
        a = a *-1

    while a > 0:
        b = a % 10
        r = r * 10 + b
        a = a // 10
    if X < 0:
        r = r *-1
    return r

File: responses/RVM3S9-UP8/reverse_digits.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(X) {
    // write your code in JavaScript
    X_str = X.toString();
    X_str_reverse = X_str.split("").reverse().join("");
    X_reverse = parseInt(X_str_reverse);
    return X_reverse
}
